vtex-sites/nextjs.store
# Feat: add attachments to cartitem

## What's the purpose of this pull request?

Updates the starter to take attachments into account when calculating the item ID: https://github.com/vtex/faststore/pull/1304

## How does it work?

The implementation uses the same logic as [FastStore API](https://github.com/vtex/faststore/blob/413494313171c200b247974b1b4c50b4a3c9a63f/packages/api/src/platforms/vtex/resolvers/validateCart.ts#L38)'s `getId` function

## How to test it?

Add items to the cart and check the browser storage (indexed db). It should work just like before, but with id separator as `::` instead of the previous `:` (this change was introduced to match the logic inside FastStore API)

## Checklist

<em>You may erase this after checking them all ;)</em>



- PR description
